src/commands/database-size.ts

```diff
@@ -0,0 +1,28 @@
+import { DatabaseSize } from "../database/size";
+
+import * as Command from "../command";
+import * as requireInstance from "../requireInstance";
+import * as requirePermissions from "../requirePermissions";
+import * as utils from "../utils";
+import * as lodash from "lodash";
+
+module.exports = new Command("database:size <path>")
+  .description(
+    "esimate the size of the Firebase subtree rooted at the specified path " +
+      "by recursively listing the sub-tree rooted at <path>"
```

Comment:
I think we should warn the developer that this downloads the whole node, and maybe increase bill.